### PR TITLE
docs: getting started using the player studio 

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

Getting started guide 

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Getting Started
```
Comment (dan-ziv) [inline docs/getting-started.md]: Change the title
Getting Started - KMC to Embed

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<head>
+<body>
+  <div id="my-player" style="width: 640px;height: 360px"></div>
+  <script type="text/javascript" src="http://www.kaltura.com/p/2196781/embedPlaykitJs/uiconf_id/41483031"></script>
+  <script type="text/javascript">
+    try {
+      var kalturaPlayer = KalturaPlayer.setup({targetId: "my-player", provider: {partnerId: 2196781, uiConfId: 41483031}});
```
Comment (dan-ziv) [inline docs/getting-started.md]: not readable - use indentation

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Getting Started
+
+### Table of Contents
+ - [Create Your Own Kaltura Player](#create)
+ - [Generate Kaltura Player Embed Code](#generate)
+ - [Embed the Kaltura Player into Your Website](#embed)
+ - [Config the Player ](#config)
+ 
+To getting started with Kaltura Player you should have an active Kaltura Management Console (KMC) account. For more Information see [here](https://corp.kaltura.com/Products/Video-Applications/Kaltura-Video-Management-Console).
```
Comment (OrenMe) [inline docs/getting-started.md]: Before getting started...

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## Create Your Own Kaltura Player <a name="create"></a>
+
+Go to [KMC](https://kmc.kaltura.com/index.php/kmc/kmc4#studio%7Cuniversal_studio) and select the *Studio* tab.
+![kmc](./images/kmc.png)
+
+Once you are in the *Studio* go to the *TV Platform Studio*.
+![studio](./images/studio.png) 
+
+Here you can see your created player list or to add a new one.<br>
```
Comment (OrenMe) [inline docs/getting-started.md]: Here you can view your existing players list and also create new players

**Pull request merged**